src/main/java/seedu/address/model/person/Person.java

```diff
@@ -86,6 +86,15 @@ public Policy getPolicy() {
         return policy;
     }
 
+    /**
+     * Compares the policy issue date to the policy expiration date to see which comes first.
+     * Returns more than 0 if expiry date is later and less than 0 otherwise
+     * @return int
+     */
+    public int comparePolicyDates() {
+        return this.policy.getPolicyExpiryDate().compareTo(this.policy.getPolicyIssueDate());
```

Comment:
Perhaps this comparison should be implemented in the `Policy` class instead? Seems like a possible violation of the Law of Demeter